src/components/Proteome/Summary/UniProtDescription/index.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+
+import { createSelector } from 'reselect';
+
+import loadData from 'higherOrder/loadData/ts';
+import { Params } from 'higherOrder/loadData/extract-params';
+
+import Loading from 'components/SimpleCommonComponents/Loading';
+
+import config from 'config';
+
+interface LoadedProps extends LoadDataProps<UniProtProteomesPayload> {}
+
+const UniProtDescription = ({ data }: LoadedProps) => {
+  if (!data) return null;
+  const { loading, payload } = data;
+  if (loading || !payload) return <Loading />;
+  return <div>{data.payload?.description}</div>;
```

Comment:
Handle cases where no description is available (e.g. UP000002473) and add a "Description" header, like we have for entries:

```suggestion
  return data.payload?.description ? (
    <>
      <h4>Description</h4>
      <p>{data.payload?.description}</p>
    </>
  ) : null;
```

Reply:
Fair enough